Description: ensure that Makefile defines a proper vpnc version, including SVN and Debian revision
 mk-version needs to be called inside an svn checkout (or full git-svn clone)
 in order to report the correct SVN revision.  Since we have this info in
 debian/changelog, take it from there instead.
Author: Florian Schlichting <fsfs@debian.org>
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -57,7 +57,7 @@
 CRYPTO_OBJS = $(addsuffix .o,$(basename $(CRYPTO_SRCS)))
 BINOBJS = $(addsuffix .o,$(BINS))
 BINSRCS = $(addsuffix .c,$(BINS))
-VERSION := $(shell sh mk-version)
+VERSION=$(shell dpkg-parsechangelog | sed -rne 's,^Version: ([^ ]+).*,\1,p')
 RELEASE_VERSION := $(shell cat VERSION)
 
 CC ?= gcc
